data/human/hails.txt

```diff
@@ -3279,6 +3636,7 @@ phrase "hostile militia"
 		"Escape from this pirate"
 		"Activate your hyperdrives"
 		"Escape and allow us to fight this battle"
+		"We will hold them off"
```

Comment:
Just "Quickly, merchants" sounds like an incomplete advice. How about "Quicky, merchants! Flee! We will hold them off".